packages/calculator-bn.js/rollup.config.js

```diff
@@ -0,0 +1,5 @@
+import { createRollupConfigs } from '../../scripts/rollup/config';
+
+import pkg from './package.json';
+
+export default createRollupConfigs({ pkg });
```

Comment:
```suggestion
export default createRollupConfigs({ pkg, config: { external: 'bn.js' } });
```

Try this along with adding `bn.js` to `peerDependencies`.

Reply:
done 8a0897407e92928c7d5b111c8d78d6ef02119882 daf2334ed691430a03d8580a9e675f9d462b38ca